### PR TITLE
Update to allow SSL to work.

### PR DESCRIPTION
**Pull request by charliemblack**
Put in the flag to allow any node 10 to work.

### CODE REVIEW

**Review by pdxcodemonkey — CHANGES_REQUESTED** [review]
This will no longer run locally in our dev or CI environments.  Can you please modify startGemFire/shutdownGemFire scripts to start server/locator with SSL enabled and modify the book-service code somehow to support switching cert files so we can run it locally?

**Comment by mmartell** [comment]
Maybe just use an environment variable (PCC-ENABLED) to allow running in PCC as well as locally and in CI. Select the system's truststore when PCC-ENABLED is true.

**Comment by mmartell** [comment]
Also noticed that the README.md has 10.16.3 as minimum version on Node.js, whereas 10.0 is minimum version in package.json.

**Comment by charliemblack** [comment]
Mike - I like the environ variable idea.    Node has a
`NODE_ENV=production` when it's running on cloud foundry.   The default is
considered development.

Charlie

**Review by karensmolermiller — COMMENTED** [review]
I don't really think that the prose that explains how to spin up a Cloud Cache cluster is necessary anymore. It existed because the example did not work when TLS encryption was enabled, so we wanted to make extra sure that users spun up a non-TLS encrypted cluster. 

I suggest accepting this PR and then asking Dave Barnes (@davebarnes97) to look at the README.

File: book-service/README.md [hunk 1]
```diff
@@ -211,19 +211,19 @@ This section uses the following names - if your Cloud Cache instance uses differ
 1. Create a Cloud Cache service instance that disables TLS encryption. For example:
```
Comment (karensmolermiller) [inline book-service/README.md:211]: Change "disables" to "enables"

Comment (charliemblack) [reply book-service/README.md:211]: done

**Comment by charliemblack** [comment]
looks like the review is still thinking I need changes - but I don't see anything to resolve.    I believe I resolved all open issues.

**Review by pdxcodemonkey — APPROVED** [review]

**Pull request merged**